wgsl/index.bs

```diff
@@ -8603,6 +8610,30 @@ path: syntax/location_attr.syntax.bs.include
 
 </table>
 
+<pre class=include>
+path: syntax/blend_src_attr.syntax.bs.include
```

Comment:
This file is missing from the PR.

Reply:
Done. Thanks!

However the checks still fail and I don't know why. Could you help me?